graphrag_sdk/orchestrator/execution_plan.py

```
@@ -1,21 +1,43 @@
 from json import loads
+from typing import Union
 from graphrag_sdk.orchestrator.step import PlanStep, StepBlockType
```

Ruff

`graphrag_sdk.orchestrator.step.StepBlockType` imported but unused

Remove unused import: `graphrag_sdk.orchestrator.step.StepBlockType`

(F401)